src/selfsign.ml

```diff
@@ -25,11 +25,9 @@ let info =
   let doc = "generate a self-signed certificate" in
   let man = [ `S "BUGS"; 
               `P "Submit bugs at https://github.com/yomimono/ocaml-certify";] in
-  Term.info "selfsign" ~doc ~man
+  Term.info "selfsign" ~version:"0.0.1" ~doc ~man
```

Comment:
Instead of `"0.0.1"` you can use `Certify_version.currentversion` (which is generated by the Makefile)